fandogh_cli/fandogh_client/__init__.py

```diff
@@ -86,6 +91,9 @@ def get_image_build(image_name, version, token):
 
 
 def create_version(image_name, version, workspace_path):
+    workspace_size = os.path.getsize(workspace_path) / 1048576  # which is 1024 * 1024
```

Comment:
this is not true. why don't you consider the .dockerignore file?

Reply:
check the size of created zip file

Reply:
from issue
`Ideally we should give some warning to the user if the workspace size is more than usual. e.g. > 20MB
and give him some hint how to use .dockerignore feature.`

so we don't want to stop uploading just some warnings a hints